README (2).md

```diff
@@ -0,0 +1,49 @@
+![PollCord_Logo](https://user-images.githubusercontent.com/36363608/193755078-ac3949c0-0db9-4697-bf99-cc62e3788621.png)
+
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/)
+
+[![Python](https://img.shields.io/badge/python-v3.7+-green.svg)](https://docs.python.org/3/)
+#    PollCord
+
+### What is PollCord?
```

Comment:
This is redundant since we already have the logo with PollCord on it, and the repo name is PollCord. So you can remove this and say `Purpose` or `Overview`.